src/modm/platform/timer/stm32/general_purpose_base.hpp.in

```diff
@@ -275,11 +275,11 @@ protected:
 	signalToChannel()
 	{
 		modm::platform::detail::SignalConnection<Signal, p>{};
-		if constexpr (Signal::Signal == Gpio::Signal::Ch1) {
+		if constexpr ((Signal::Signal == Gpio::Signal::Ch1) || (Signal::Signal == Gpio::Signal::Ch1n)) {
```

Comment:
This works for `g4` and maybe some other families, but not for older `f` series cpus...

Reply:
The inverted channel 4 only exists on G4, H5 and U5 devices. On others `Gpio::Signal::Ch4n` is not a valid expression.